src/acceptance-test/java/de/zalando/aruha/nakadi/repository/db/EventTypeDbRepositoryTest.java

```diff
@@ -108,7 +109,7 @@ public void whenUpdateExistingEventTypeItUpdates() throws NakadiException, Dupli
 
         repository.saveEventType(eventType);
 
-        eventType.setCategory("new-category");
+        eventType.setCategory(EventCategory.UNDEFINED);
```

Comment:
This test wasn't written in the safest way possible. This change should had broken it. The idea is that updates should indeed be persisted. We should set a different value than the original here.